README.rst

```diff
@@ -46,12 +46,18 @@ Store on Amazon S3:
 
     kinto.attachment.aws.access_key = <AWS access key>
     kinto.attachment.aws.secret_key = <AWS secret key>
-    kinto.attachment.aws.bucket = <bucket name>
+    kinto.attachment.aws.bucket_name = <bucket name>
     kinto.attachment.aws.acl = <AWS ACL permissions|public-read>
 
 
+.. note::
+
+    ``access_key`` and ``secret_key`` can be omitted when using AWS Identity and
```

Comment:
Prefer "may" to "can" here, since using IAM doesn't necessarily obviate the need for these keys.